xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
For these sorts of things do not rely on the implementation to always provide it, specify that it should be there and check.

"trust but verify"